### ISSUE TITLE
error while training

### ISSUE THREAD

**Issue by bikram42** [YerevaNN/R-NET-in-Keras#26]
ValueError: Error when checking input: expected Q_vecs to have shape (None, None, 300) but got array with shape (50, 30, 25)

**Comment by TarunTater** [comment]
@bikram42  - were you able to solve this?

**Comment by bikram42** [comment]
@TarunTater yup i was able to solve the issue, but unfortunately now i don't remember what changes i made to the code. I do have code anyways

**Comment by TarunTater** [comment]
@bikram42 - can you please tell me the repo link where you have the code. I would like to try it out and see if i can also resolve the error. Thanks a lot!

**Comment by panwarnaveen9** [comment]
@bikram42 - hi, please post the link to your working code. Even i ran through the same problem and wanted to work on this. Thanks

**Comment by bikram42** [comment]
@TarunTater @panwarnaveen9 currently i have the code on my laptop. I will upload it on google drive and share the link

**Comment by bikram42** [comment]
https://drive.google.com/open?id=1MBxcEbKgU1rNkniEmNSDdqeQKeW2oFdm

this is the google drive link for the code.  Install keras 2.0.6. I have not included the data folder.
